app/views/support_requests/index.html.erb

```diff
@@ -1,87 +1,81 @@
-                <%= form_with(model: support_request,
-                              local: true,
-                              class: "depot_form") do |form| %>
-                    <div class="filed">
-                        <%= form.label :response, "Write Response" %>
-                        <%= form.rich_text_area :response, id: :support_request_response %>
-                    </div>
-                <div class="actions">
-                    <%= form.submit "Send Response" %>
-                </div>
-            <% end %>
-            <% else %>
-                <h4>Our response:</h4>
-                <p>
-                    <blockquote>
-                        <%= support_request.response %>
-                    </blockquote>
-                </p>
-            <% end %>
-
-            <hr>
-        </li>
-    <% end %>
+          <dt>Email</dt>
+          <dd class="ml-4"><%= support_request.order.email %></dd>
+          
+          <dt>Address</dt>
+          <dd class="ml-4"><%= support_request.order.address %></dd>
+          
+          <dt>PayType</dt>
+          <dd class="ml-4"><%= support_request.order.pay_type %></dd>
+          
+          <dt>Line Items</dt>
+          <dd>
+            <ul class="ml-4 list-disc">
+              <% support_request.order.line_items.each do |line_item| %>
+                <li>
+                  <%= line_item.product.title %>
+                  (<%= line_item.product.price %>) 
+                </li>
+              <% end %> 
+            </ul>
+          </dd> 
+        </dl>
+      <% else %>
+        <h3 class="notice">No associated order</h3>
+      <% end %>
+      <% if support_request.response.blank? %>
+        <%= form_with(model: support_request, local: true, class: "depot_form") do |form| %> 
+          <div class="field">
+            <%= form.label :response, "Write Response" %>
+            <%= form.rich_text_area :response, id: :support_request_response %> </div>
+          <div class="actions">
+            <%= form.submit "Send Response" %>
+          </div>
+        <% end %>
+      <% else %>
+        <h4>Our response:</h4>
+        <p>
+          <blockquote>
+            <%= support_request.response %>
+          </blockquote>
+        </p>
+        <% end %>
+        <% if support_request.response.blank? %>
+          <%= form_with(model: support_request,
+                        local: true,
+                        class: "depot_form") do |form| %> 
+            <div class="field">
+              <%= form.label :response, "Write Response" %>
+              <%= form.rich_text_area :response, id: :support_request_response %> </div>
+            <div class="actions">
+              <%= form.submit "Send Response" %>
+            </div>
+          <% end %>
+          <% else %>
+            <h4>Our response:</h4> 
+            <p>
+              <blockquote>
+                <%= support_request.response %> 
+              </blockquote>
+            </p>
```

Comment:
This code is duplicated and can be removed. This will fix the duplicated form problem.
```suggestion
```